system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -374,6 +375,13 @@ class CLdapConfig : public CInterface, implements ILdapConfig
             LdapUtils::normalizeDn(dnbuf.str(), m_basedn.str(), m_filescope_basedn);
 
         dnbuf.clear();
+        cfg->getProp(".//@viewsBasedn", dnbuf);
+        if(dnbuf.length() == 0)//TODO REMOVE THIS WHEN CONFIGMGR SETS viewsBasedn
```

Comment:
Maybe this can be removed now? Since https://github.com/hpcc-systems/HPCC-Platform/pull/8932 is about to get merged...